WordPress/src/main/java/org/wordpress/android/ui/barcodescanner/BarcodeScanner.kt

```diff
@@ -0,0 +1,99 @@
+package org.wordpress.android.ui.barcodescanner
+
+import android.content.res.Configuration
+import android.util.Size
+import androidx.camera.core.CameraSelector
+import androidx.camera.core.ImageAnalysis
+import androidx.camera.core.ImageAnalysis.STRATEGY_KEEP_ONLY_LATEST
+import androidx.camera.core.ImageProxy
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.camera.view.PreviewView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.viewinterop.AndroidView
+import androidx.core.content.ContextCompat
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+import org.wordpress.android.ui.compose.theme.AppTheme
+import androidx.camera.core.Preview as CameraPreview
+
+@Composable
+fun BarcodeScanner(
+    codeScanner: CodeScanner,
+    onScannedResult: (Flow<CodeScannerStatus>) -> Unit
+) {
+    val context = LocalContext.current
+    val lifecycleOwner = LocalLifecycleOwner.current
+    val cameraProviderFuture = remember {
+        ProcessCameraProvider.getInstance(context)
```

Comment:
TIL I don't think I ever used this before 👍

Reply:
Same here. Thank @AnirudhBhat  🙇